### PR TITLE
Update diagrams and release-process description

### PR DESCRIPTION
**Pull request by schmie**
What
----
- Remove colour coded information from diagrams and replace with text
- Introduces git main branch as the preferred term.
- Add an enumerated description of the release process.

How to review
-------------

Review page locally. 

Who can review
--------------
not @schmie

### CODE REVIEW

**Review by kr8n3r — APPROVED** [review]
looks good

File: source/team/working_practices.html.md [hunk 1]
```diff
@@ -31,14 +33,31 @@ The development process consists of the following steps:
...
 
 [Full size version](/diagrams/release-sequence.svg)
 
+The release process consists of the following steps:
+
+1. Developers (engineers) commit code to a development branch on a git repository.
+
+2. When the development is complete, a developer raises a pull request against the main branch of the git repository for review.
+
+3. If the review is passed, the reviewer merges the changes into the main branch.
```
Comment (cmcnallygds) [inline source/team/working_practices.html.md:47]:
```suggestion
3. Another team member reviews the pull request and if there are no problems, merges the changes into the main branch.
```

Comment (cmcnallygds) [inline source/team/working_practices.html.md:47]: Is it worth also stating what happens if there is an issue with something in the PR?

File: source/team/working_practices.html.md [hunk 2]
```diff
@@ -31,14 +33,31 @@ The development process consists of the following steps:
...
+
+4. A new commit to the main branch triggers a Concourse git_resource that perforns an automated deployment to the staging environment.
```
Comment (cmcnallygds) [inline source/team/working_practices.html.md:49]:
```suggestion
4. A new commit to the main branch triggers a Concourse git_resource that performs an automated deployment to the staging environment.
```

File: source/team/working_practices.html.md [hunk 3]
```diff
@@ -31,14 +33,31 @@ The development process consists of the following steps:
...
 
 [Full size version](/diagrams/release-sequence.svg)
 
+The release process consists of the following steps:
+
+1. Developers (engineers) commit code to a development branch on a git repository.
+
+2. When the development is complete, a developer raises a pull request against the main branch of the git repository for review.
+
+3. Another team member reviews the pull request and if there are no problems, merges the changes into the main branch.
```
Comment (schmie) [inline source/team/working_practices.html.md:47]:
```suggestion
3. Another team member reviews the pull request and if there are no problems, merges the changes into the main branch. Alternatively, they may also provide feedback to the developers and request corrections and/or additional work.
```

File: source/team/working_practices.html.md [hunk 4]
```diff
@@ -31,14 +33,31 @@ The development process consists of the following steps:
...
+4. A new commit to the main branch triggers a Concourse git_resource that performs an automated deployment to the staging environment.
+
+5. The staging environment runs automated tests.
+  - 5 a. If the deployment or tests fail, the automated release process is halted and the team is notified.
+  - 5 b. If the deployment is successful and all tests pass, the main branch commit is tagged for release to production.
+
+6. A new release tag triggers a Concourse git_resource that performs an automated deployment to the production environment.
+
+This process does not include the use of git commit signatures which we use to
+verify developer and merger identity for the paas-cf and paas-bootstrap git
```
Comment (schmie) [inline source/team/working_practices.html.md:58]:
```suggestion
verify developer and merger identity for a number of git
```

File: source/team/working_practices.html.md (hunk 1 above)
Comment (schmie) [reply source/team/working_practices.html.md]: Resolving below

File: source/team/working_practices.html.md (hunk 3 above)
Comment (schmie) [reply source/team/working_practices.html.md:47]: Agreed on wording on Skype

File: source/team/working_practices.html.md (hunk 4 above)
Comment (schmie) [reply source/team/working_practices.html.md:58]: Not all, yet, but more than two

File: source/team/working_practices.html.md (hunk 1 above)
Comment (schmie) [reply source/team/working_practices.html.md]: Resolved/ outdated - see below

**Pull request merged**